zipline/finance/blotter.py

```diff
@@ -136,6 +140,23 @@ def cancel(self, order_id):
             # along with newly placed orders.
             self.new_orders.append(cur_order)
 
+    def execute_cancel_policy(self, event):
```

Comment:
Since we currently only call the cancel policy at day end, we could cut the extra parameter of `event`.